Lib/test/test_type_params.py

```diff
@@ -1452,3 +1452,14 @@ def f[T: (int, str)](): pass
                 self.assertEqual(annotationlib.call_evaluate_function(case.evaluate_constraints, annotationlib.Format.VALUE), (int, str))
                 self.assertEqual(annotationlib.call_evaluate_function(case.evaluate_constraints, annotationlib.Format.FORWARDREF), (int, str))
                 self.assertEqual(annotationlib.call_evaluate_function(case.evaluate_constraints, annotationlib.Format.SOURCE), '(int, str)')
+
+    def test_const_evaluator(self):
```

Comment:
this is quite a perplexing test haha. Maybe add a `# this used to segfault!` comment, so it's obvious why we have the test?